release/src/Main.hs

```diff
@@ -131,17 +143,15 @@ main = do
                 (forM_ npmPackages
                   $ \rule -> liftIO $ callCommand $ "bazel run " <> rule <> ":npm_package.publish -- --access public")
 
-         | optsLocallyInstallJars -> do
-              let lib_jars = filter (\(mvn,_) -> (artifactType mvn == "jar" || artifactType mvn == "pom") && Maybe.isNothing (classifier mvn)) mvnUploadArtifacts
-              forM_ lib_jars $ \(mvn_coords, path) -> do
-                  let args = ["install:install-file",
-                              "-Dfile=" <> pathToString releaseDir <> pathToString path,
-                              "-DgroupId=" <> (groupIdString $ groupId mvn_coords),
-                              "-DartifactId=" <> (T.unpack $ artifactId mvn_coords),
-                              "-Dversion=0.0.0",
-                              "-Dpackaging=" <> (T.unpack $ artifactType mvn_coords)]
-                  liftIO $ callProcess "mvn" args
-         | otherwise -> $logInfo "Dry run selected: not uploading, not installing"
+          | optsLocallyInstallJars -> do
+              pom <- generateAggregatePom bazelLocations mvnArtifacts
+              pomPath <- (releaseDir </>) <$> parseRelFile "pom.xml"
+              liftIO $ do
+                  T.IO.writeFile (toFilePath pomPath) pom
+                  (_, _, _, mvnHandle) <- createProcess ((proc "mvn" ["initialize"]) { cwd = Just (toFilePath releaseDir) })
```

Comment:
Please use `withCreateProcess` to make sure that resource cleanup works properly.

Reply:
Done.